### PR TITLE
Add repo setup documentation

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: howto/repo-setup.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Repository setup 
+
+We recommend that CNCF projects seperate docs into their own repository, away from code. This has the following advantages:
```
Comment (nate-double-u) [inline howto/repo-setup.md:3]:
```suggestion
We recommend that CNCF projects separate docs into their own repository, away from code. This has the following advantages:
```

**Review by nate-double-u — APPROVED** [review]
Looks good to me - just one nit inline

File: howto/repo-setup.md (hunk 1 above)
Comment (celestehorgan) [reply howto/repo-setup.md]: fixed in latest push.

**Comment by nate-double-u** [comment]
@celestehorgan, you mentioned the checklist I had been working on for the hugo-netlify-starter. That work was being done over here: https://github.com/cncf/hugo-netlify-starter/pull/31

I'm happy to open an issue on this repo and continue the work here.

**Pull request merged**